redbot/cogs/audio/core/events/lavalink.py

```diff
@@ -284,10 +284,16 @@ async def lavalink_event_handler(
                             ).format(error=description),
                         )
                     else:
+                        try:
+                            # Lavalink v3.4 jar files and later with RLL 0.9.0 and later
+                            extra_error_string = extra.get('message')
+                        except AttributeError:
+                            # Lavalink v3.3.x.x jar files and previous with RLL 0.8.x and previous
+                            extra_error_string = extra
```

Comment:
We pin our RLL version so there's no need to support both.
```suggestion
                        extra_error_string = extra.get("message")
```